srcpkgs/shadowsocks-libev/template

```diff
@@ -0,0 +1,22 @@
+# Template file for 'shadowsocks-libev'
+pkgname=shadowsocks-libev
+version=3.2.0
+revision=1
+build_style=gnu-configure
+makedepends="asciidoc xmlto pcre-devel mbedtls-devel libsodium-devel c-ares-devel libev-devel"
```

Comment:
asciidoc and xmlto are meant to be eecuted on the host system move them to hostmakedepends=